--- a/setup.py
+++ b/setup.py
@@ -50,10 +50,10 @@ setup(
     ],
     include_package_data = True,
     entry_points = entry_points,
-    extras_require = dict(
-        test=['zope.testing', 'manuel',
-              'bobo ==2.3.0', 'zdaemon', 'zc.zdaemonrecipe',
-              'zc.recipe.deployment']),
+    #extras_require = dict(
+    #    test=['zope.testing', 'manuel',
+    #          'bobo ==2.3.0', 'zdaemon', 'zc.zdaemonrecipe',
+    #          'zc.recipe.deployment']),
     zip_safe=False,
     classifiers = [
        'Intended Audience :: Developers',
